internal/tools/schema_check.sh

```diff
@@ -32,12 +32,15 @@ grep -o -e '## v[1-9].*\s' $root_dir/CHANGELOG.md | grep -o '[1-9].*' | while re
     exit 3
   fi
 
-  curl --no-progress-meter https://opentelemetry.io/schemas/$ver > verify$ver
-
-  diff verify$ver $file && echo "Published schema at https://opentelemetry.io/schemas/$ver is correct" \
-    || (echo "Published schema at https://opentelemetry.io/schemas/$ver is incorrect!" && exit 3)
-
-  rm verify$ver
+  # Schema file will no be served directly from this repository when linked
+  # into opentelemetry.io.  We disable this for now and need to move the check
+  # into the website.
```

Comment:
Because our changelog NOW includes a version not yet served. It won't be served until I tag this repository.

I opened a bug to shuffle around this logic.  Specifically, this PR would fail with `1.21.0` not being available on the website if I enabled this.

Reply:
See #191

Reply:
Going to resolve this.

TL;DR: is that previously I think we wouldn't bump the CHANGELOG.md as part of the release, leading to a release tag where CHANGELOG had an `## unreleased` header.   The release workflow I'm using now involves the tag having up-to-date SCHEMA file + `CHANGELOG`, however we are using the presence of a version in CHANGELOG to check the opentelemetry.io hosting.   There will now be a delay between cutting a release, bumping the CHANGELOG and opentelemetry.io hosting that previously we didn't have.

We'll figure out a new mechanism for testing this out as part of #191